docs/docs/concepts/workflows.rst

```diff
@@ -1,35 +1,238 @@
+   :class: no-scroll
+
+   * - Operation Type
+     - Resource Owners Team
+     - Requestors Team
+     - All Approvers
+     - Admin
+   * - New Topic Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Update Topic Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Delete Topic Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Claim Topic Request
+     - ✅
+     -
+     - ✅
+     -
+   * - Promote Topic Request
+     -
+     - ✅
+     - ✅
+     -
+
+Connector owner
+---------------------
+
+Klaw can integrate with Kafka Connect and define various types of connectors. Any team can request a specific connector, and they will become the owners of that connector.
+
+
+Connector request approvals
+`````````````````````````````
+The table below outlines the roles responsible for approving various connector-related requests:
+
+.. list-table:: Connector Approver matrix
+   :header-rows: 1
+   :class: no-scroll
+
+   * - Operation Type
+     - Resource Owners Team
+     - Requestors Team
+     - All Approvers
+     - Admin
+   * - New Connector Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Update Connector Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Delete Connector Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Claim Connector Request
+     - ✅
+     -
+     - ✅
+     -
+   * - Promote Connector Request
+     -
+     - ✅
+     - ✅
+     -
+
+Schema owner
+---------------
+
+Schema Owner
+  Klaw can integrate with Karapace or Schema registry and define schemas on a topic. Currently, the topic owner team can raise a schema request, and they become the owners of the schema and its evolution.
+
+
+Schema request approvals
+``````````````````````````
+
+The table below outlines the roles responsible for approving various schema-related requests:
+
+.. list-table:: Schema Approver matrix
+   :header-rows: 1
+   :class: no-scroll
+
+   * - Operation Type
+     - Resource Owners Team
+     - Requestors Team
+     - All Approvers
+     - Admin
+   * - New Schema Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Delete Schema Request
+     -
+     - ✅
+     - ✅
+     -
+   * - Promote Schema Request
+     -
+     - ✅
+     - ✅
+     -
+
+
+.. note::
+   With Schemas there is no `Claim Schema` as it is assigned to a topic and so the Topic owner owns the Schema. Likewise there is no `Update Schema` as existing Schemas are kept and a new Schema with an incremented version is added instead.
+
+Subscription owner
+----------------------
+
+Subscription Owner
+  In Klaw, subscriptions (ACLs) are defined to secure Kafka topics. Each team that wants to produce or consume from a topic becomes the subscription owner. Teams can submit subscription requests for a specific topic (either as a producer or consumer), which are then reviewed by the Topic Owner team for approval. The Topic Owner team alone can delete subscriptions, access consumer offsets, view topic contents, and manage any relevant subscription credentials, if applicable.
+
```

Comment:
The Topic Owner team alone can delete subscriptions.
This is actually a mistake on my part
Only the owner of the subscription can delete the subscription.